### ISSUE TITLE
Missing masks

### ISSUE THREAD

**Issue by tarashakhurana** [y-zheng18/point_odyssey#4]
Hi, thanks for the amazing one-of-a-kind dataset! I downloaded and extracted the dataset but couldn't find the masks for this sequence `train/cnb_dlab_0225_3rd`. Could you verify on your end?

**Comment by y-zheng18** [comment]
Yes it seems the masks in that sequence are missing. Will update soon. Thanks!

**Comment by tarashakhurana** [comment]
Thanks for confirming! Additionally, I wanted to ask if there is a way to add semantic classes to the instance masks? For example, even if I am able to get the distinction between walls and other foreground objects, it'll be a huge win for me. Do you know of an easy way to get this?

**Comment by y-zheng18** [comment]
I'm sure it's possible since a mask color corresponds to a unique asset. You can refer to [here](https://github.com/y-zheng18/point_odyssey/blob/main/utils/gen_tracking.py#L226) to see how to find the correspondences.

**Comment by m43** [comment]
Hej @y-zheng18, it seems that some of the datapoints do not have `trajs_3d` annotations but have `array([None], dtype=object)` instead. Do you happen to know why and if they could be added to the data? The following training data scenes are affected: `character0_`, `character0_f2`, `character2_f`, `character5`, `gso_in_big`.

**Comment by y-zheng18** [comment]
yes in those scenes the trajs_3d annotations are missing. We are fixing that and will update the dataset soon.

**Comment by saunair** [comment]
Hi @y-zheng18 I noticed the same issue with the trajs missing, do you have a version of the dataset where it's fixed?

**Comment by y-zheng18** [comment]
Hi @saunair, did you try the [v1.2](https://drive.google.com/drive/u/1/folders/1W6wxsbKbTdtV8-2TwToqa_QgLqRY3ft0) data?

**Comment by saunair** [comment]
Hi @y-zheng18 Thanks for your reply!
Yes, this is with v1.2. I notice these scenes (not exclusvie), don't have the trajs3d key in them
```character4
character5_f
character6_f
character4_f
character3
character0_f2
character0_f
character0
character2
character5
character3_f
character1_f
character1
```

**Comment by saunair** [comment]
Hi @y-zheng18 I'm unsure if this is an issue caused due to extraction, as I only used the huge train.gz file in drive to get the train folders. When I look through the scenes I have, only see 104 scenes (I expected more). 
I didn't use the .aa, .bb files, thinking the single big file would have all the necessary files. Is that assumption correct?

**Comment by y-zheng18** [comment]
It should be good. For `character*` sequences we don't have trajs3d annotations. To make up for that, in v1.2 we render more sequences like `r0_new_, r0_new_f` using similar assets.
